framework/integration/init.go

```diff
@@ -0,0 +1,146 @@
+// +build k8srequired
+
+package integration
+
+import (
+	"github.com/giantswarm/e2e-harness/pkg/harness"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/operatorkit/framework"
+	"github.com/giantswarm/operatorkit/framework/integration/test_resource_set"
+	"github.com/giantswarm/operatorkit/informer"
+)
+
+const (
+	namespace = "finalizer-integration-test"
+)
+
+var (
+	err error
+
+	k8sClient     kubernetes.Interface
+	k8sRESTClient rest.Interface
+)
+
+func init() {
+	k8sClient, err = newK8sClient()
+	if err != nil {
+		panic(err)
+	}
+	k8sRESTClient, err = newK8sRESTClient()
+	if err != nil {
+		panic(err)
+	}
+}
+
+func newK8sClient() (kubernetes.Interface, error) {
+	config, err := clientcmd.BuildConfigFromFlags("", harness.DefaultKubeConfig)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	k8sClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	return k8sClient, nil
+}
+
+func newK8sRESTClient() (rest.Interface, error) {
+	return k8sClient.CoreV1().RESTClient(), nil
+}
```

Comment:
I don't think it's a good idea to make this a separate function. Caller must be sure that k8sClient is initialized and it isn't immediately visible.